### PR TITLE
Updating sentieon to 202308.02

### PR DESCRIPTION
**Pull request by asp8200**
Updating Sentieon to version 202308.02.

I don't expect that the CI-tests will run, since - as far as I know - nf-core's test-license server for sentieon is still down.

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/sentieon/applyvarcal/environment.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::sentieon=202308.01
+  - bioconda::sentieon:202308.02
```
[the remark this answers is not in the document]
Comment (asp8200) [reply modules/nf-core/sentieon/applyvarcal/environment.yml:7]: 🤦‍♂️

**Review by SPPearce — APPROVED** [review]

**Comment by asp8200** [comment]
@SPPearce @maxulysse : Could we and should we merge without the tests having been run?

**Comment by asp8200** [comment]
I'll run some tests of at least some of these Sentieon modules over in Sarek - using a real Sentieon license.

**Comment by asp8200** [comment]
The tests are running fine over in Sarek:   https://github.com/nf-core/sarek/pull/1487

**Pull request merged**